app/src/main/res/values/strings.xml

```diff
@@ -92,15 +92,15 @@
     <string name="settings_dark_theme">Dark</string>
     <string name="settings_key_dark_theme" translatable="false">dark</string>
     <string name="settings_key_max_font_size_scale" translatable="false">pref_max_font_size_scale</string>
-    <string name="settings_max_font_size_scale">Maximum font size scale</string>
+    <string name="settings_max_font_size_scale">Max font size</string>
```

Comment:
I might be mistaken here, but this is how I understood it.

Reply:
It's kinda how big the font size can be compared to the default limits I have set for small, medium or large parts of text. 100% means you get it as I coded it, lower percentages means the fonts will be smaller and higher means they will be bigger. Maybe "Font size scale" makes more sense, as it's a percentage:

![image](https://user-images.githubusercontent.com/1885159/113523141-4b938180-95a6-11eb-82d0-41f74719db30.png)

Reply:
"get it as you coded it", size or limits?
"lower percentages means the fonts will be smaller and higher means they will be bigger" than? when? Does it have to do with system settings?

Reply:
It is technically the upper bound. If there is not enough space to reach the upper bound set it will not reach it but it will use the highest size that still fits.

Reply:
To me that seems like pointing out ~"the text can't be bigger than what can be displayed, even if you select max size". I don't see an additional point made.

Reply:
I'm not trying to make a point, just was trying to answer your question